### DISCUSSION TITLE
RAK811 as Gateway

### DISCUSSION THREAD

**Discussion by klaids** [AmedeeBulle/pyrak811#23] [Q&A]
Goodmorning,
I am running some test with [IoT LoRa Node pHAT for Raspberry Pi](https://uk.pi-supply.com/products/iot-lora-node-phat-for-raspberry-pi) in the mode lora p2p. Is it possibile to configure the module with library as gateway lorawan TTN?
Or it's possibile to configure only as node and so, if I would to send a package with lorawan i must have a gateway in the TTN near me. 
Best Regards

**Comment by AmedeeBulle** [comment]
Hi,

The LoRa node is a "node only", you cannot use it as a gateway.

You are correct, you will only be able to use it as a TTN node if there is a TTN Gateway in your area.
